clipping_manager/filters.py

```diff
@@ -9,7 +9,7 @@ class ClippingFilter(django_filters.FilterSet):
     """
```

Comment:
Random comment to the feature: I wonder whether the single-clipping-deletion is what people are looking for, or whether they rather need a batch deletion option. 🤔 
On the other hand, batch deletion might be mostly used for deleting whole books, so enabling them to delete books next to this single-clipping deletion might be perfect 🤷 

Having said that, maybe it's smart to quickly add our delete method to the django admin as a separate soft-delete action, then I can directly delete more clippings - because I get mails asking for it every now and then 🙈

Reply:
Single-clipping-deletion is much safer and, in combination with book-deletion, would provide perfect balance between utility and preventing unwanted actions.
I would make this book-deletion a separate issue, though

> Having said that, maybe it's smart to quickly add our delete method to the django admin as a separate soft-delete action, then I can directly delete more clippings - because I get mails asking for it every now and then 🙈

Sure, it seems resendable. I'll try to find out how to add a model method to the django admin.

Reply:
> in combination with book-deletion, would provide perfect balance between utility and preventing unwanted actions.

True!